_posts/0000-01-02-simbatski.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
TRAIN YOUR MIND TO FOCUS, AND YOUR BRAINS WON"T COMPLAIN

![image](https://user-images.githubusercontent.com/57085127/86342679-9e7b8980-bc60-11ea-9d33-92a5b6d679c4.png)